index.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert')
+const path = require('path')
+const walk = require('walk-sync')
+const { chain } = require('lodash')
+const { supportedVersions } = require('./package.json')
+const contentDir = path.join(__dirname, 'content')
+const allPages = walk(contentDir, { directories: false })
+  .filter(filename => filename.endsWith('.md'))
+  .map(filename => {
+    const fullPath = path.join(contentDir, filename)
+    const nodeVersion = filename.split(path.sep)[0]
+    const locale = filename.split(path.sep)[1]
+    const filePath = filename
+      .split(path.sep)
+      .slice(2)
+      .join(path.sep)
+    return { locale, nodeVersion, filePath, fullPath }
+  })
+
+async function getPages (nodeMajor, locale) {
```

Comment:
what is the reasoning behind `getPages` being async? Just to be future proof?

Reply:
I think, better to declare it as `async` from a start, than later, when we should make some asynchronous stuff inside to not break backward compatibility

Reply:
Yep that was my thinking.